src/context/viewer-context.tsx

```diff
@@ -165,14 +168,26 @@ function useAuthedViewer() {
     window.becomeUser = auth.becomeUser
   }, [])
 
+  const handleAccessTokenAuthentication = async (
+    authToken: string,
+    expiresInSeconds: string,
+  ) => {
+    const authenticatedUser = await auth.handleAccessTokenAuthentication(
+      authToken,
+      expiresInSeconds,
+    )
+    setViewer(authenticatedUser)
+  }
```

Comment:
This wraps `Auth`'s `handleAccessTokenAuthentication` so that the user data it returns gets set as the viewer in the viewer context. This was necessary to get both the main view and the header to recognized the user corresponding to the received auth token.